src/platform/mhv/hooks/useDatadogRum.jsx

```diff
@@ -3,7 +3,34 @@ import { datadogRum } from '@datadog/browser-rum';
 
 import { environment } from '@department-of-veterans-affairs/platform-utilities/exports';
 
+const getCookie = name => {
```

Comment:
TESTING:  I think we should have some sort of unit test around this function. It seems to the core of the logic, so some sort of test would be good. Even if it's just the happy path and few sad paths

Reply:
good point. Do you recommend exporting this function from some util class?